local/compose/build.go

```diff
@@ -98,7 +98,12 @@ func (s *composeService) ensureImagesExists(ctx context.Context, project *types.
 
 	}
 
-	err := s.build(ctx, project, opts, "auto")
+	mode := "auto"
+	if quietPull {
+		mode = "quiet"
+	}
+
+	err := s.build(ctx, project, opts, mode)
```

Comment:
I would not mix mode (auto/tty/...) with this pull-specific option. This will lead to unwanted side effects or hard to understand/evolve code. I'd prefer to have a separate param here, a lot more explicit

Reply:
I understand your PoV but on the other hand, as `mode` is responsible to manage output, using `quiet` here sounds legitimate. Maybe we could contribute this special value to https://github.com/docker/buildx/blob/080e9981c7332513158f0deff7d4057c370395d7/util/progress/printer.go ?

Reply:
@ndeloof PTL https://github.com/docker/buildx/pull/558

Reply:
👍